p4-16/spec/P4-16-draft-spec.mdk

```diff
@@ -1176,7 +1176,7 @@ produced by different programmers at different times:
 Similar to the C standard library, the P4 language specification
 defines a core library that declares useful P4 constructs. A
 description of the P4 core library is provided in Appendix
-[#sec-p4-core-lib]. Most P4 programs will include the core
+[#sec-p4-core-lib]. All P4 programs must include the core
```

Comment:
Does the `checkCorelibMethods` helper in the type checker validate *all* of the elements of `core.p4`? Or do we need to open an issue on `p4c`?